packages/video_player/video_player/lib/video_player.dart

```diff
@@ -282,6 +283,27 @@ class VideoPlayerController extends ValueNotifier<VideoPlayerValue> {
         package = null,
         super(const VideoPlayerValue(duration: Duration.zero));
 
+  /// Constructs a [VideoPlayerController] playing a network video.
+  ///
+  /// The URI for the video is given by the [dataSource] argument.
+  ///
+  /// **Android only**: The [formatHint] option allows the caller to override
+  /// the video format detection code.
+  ///
+  /// [httpHeaders] option allows to specify HTTP headers
+  /// for the request to the [dataSource].
+  VideoPlayerController.networkUrl(
```

Comment:
Can we use` VideoPlayerController.networkUri` here?

Reply:
Considering we are passing in a Uri. 
After checking `canLaunchUrl` ,  `networkUrl` works

Reply:
> Considering we are passing in a Uri.

URLs are URIs. I can't think of any circumstance where passing a non-URL URI here would be valid.